aspnet/PivotClient/OLAP-Getting-Started.md

```diff
@@ -1,14 +1,14 @@
 ---
 layout: post
-title: OLAP Getting Started | PivotClient | ASP.NET | Syncfusion
-description: olap getting started
+title: OLAP Getting Started with ASP.NETWEBForms PivotClient | Syncfusion
```

Comment:
OLAP Getting Started with ASP.NET Web Forms PivotClient | Syncfusion - change like this

Reply:
changed